From 1069a4f2bbc3d2aba2984d64eb19c73f20aec500 Mon Sep 17 00:00:00 2001
From: Nikolaus Rath <Nikolaus@rath.org>
Date: Tue, 23 Feb 2016 16:14:11 -0800
Subject: Ignore compiler warnings due to old Cython version

Origin: debian
Forwarded: not-needed
Last-Update: <2016-02-23>
Author: Nikolaus Rath <Nikolaus@rath.org>

Cython versions prior to 0.24 generate Gcc compiler warnings.
Since Cython 0.24 isn't in Debian yet, we ignore these warnings.
---
 setup.py | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/setup.py b/setup.py
index cc21a..16cc0 100755
--- a/setup.py
+++ b/setup.py
@@ -80,9 +80,6 @@ def main():
     # We may have unused functions if we compile for older FUSE versions
     compile_args.append('-Wno-unused-function')
 
-    # Value-changing conversions should always be explicit.
-    compile_args.append('-Werror=conversion')
-
     # Note that (i > -1) is false if i is unsigned (-1 will be converted to
     # a large positive value). We certainly don't want to do this by
     # accident.
